test/test_routes.py

```diff
@@ -400,12 +400,15 @@ def test_mount_gradio_app_raises_error_if_event_queued_but_queue_disabled(self):
 
     def test_proxy_does_not_leak_hf_token_externally(self):
         gr.context.Context.hf_token = "abcdef"
-        r = routes.App.build_proxy_request(
-            "https://gradio-tests-test-loading-examples-private.hf.space/file=Bunny.obj"
-        )
-        assert "authorization" in dict(r.headers)
-        r = routes.App.build_proxy_request("https://google.com")
-        assert "authorization" not in dict(r.headers)
+        app = routes.App()
+        interface = gr.Interface(lambda x: x, "text", "text")
+        app.configure_app(interface)
+        with pytest.raises(ValueError):
+            app.build_proxy_request(
+                "https://gradio-tests-test-loading-examples-private.hf.space/file=Bunny.obj"
+            )
+        with pytest.raises(ValueError):
+            app.build_proxy_request("https://google.com")
```

Comment:
These are probably checking for the wrong exception now.

I think it would also be worth it to have a test that a proxy request _can_ be built when `root_urls` would allow it?

Reply:
> These are probably checking for the wrong exception now.

Thanks, will fix!

> I think it would also be worth it to have a test that a proxy request can be built when root_urls would allow it?

There's a different test that tests this: `test/test_external.py::TestLoadInterface::test_loading_files_via_proxy_works`